src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_overview.md

```diff
@@ -23,9 +23,18 @@ The articles in this chapter describe the particular approaches used in the Blan
 -  [Create a responsive mobile theme based on Blank]
 
 ## Terms used {#fedg_rwd_terms}
-  Term           Description
-  -------------- ---------------------------------------------------------------------------------
-  *Breakpoint*   The width of the user's screen that causes your responsive layout to change.
+<table>
```

Comment:
Hi @abrarpathan19. Markdown is the preferred way for organizing the page layout (including tables). Please, use Markdown to create a table instead of raw HTML. 

Thank you.

Reply:
Hello @rogyar, Thanks for the Suggestions